include/nbl/builtin/hlsl/bxdf/ndf/ggx.hlsl

```diff
@@ -278,7 +278,11 @@ struct GGX
         isInfinity = hlsl::isinf(D);
         quant_type dmq;
         if (isInfinity)
+        {
+            dmq.microfacetMeasure = scalar_type(0.0);
+            dmq.projectedLightMeasure = scalar_type(0.0);
```

Comment:
both should be set to `bit_cast<scale_type>(numeric_limits<scale_type>::inf)` not 0s